assets/js/xrp-ledger-toml-checker-test.js

```diff
@@ -0,0 +1,164 @@
+// This code takes in a wallet address, checks the domain field, then gets the TOML info to verify the domains ownership.
+// This is runnable in NODE JS for easier testing, and works the same as the code in xrp-ledger-toml-checker.js
+const WebSocket = require('ws');
+const https = require('https');
+const TOML = require('../vendor/iarna-toml-parse');
+
+const TOML_PATH = "/.well-known/xrp-ledger.toml"
+const TIPS = 'Check if the file is actually hosted at the URL above, check your server\'s HTTPS settings and certificate, and make sure your server provides the required CORS header.\n'
+const TIPS_1 = 'Make sure you are entering a valid XRP address.\n'
+const TIPS_2 = 'Make sure the wallet address has the domain field.\n'
+
+const ACCOUNT_FIELDS = [
+    "address",
+    "network",
+    "desc"
+]
+
+// set 'wallet' to any of these test wallets
+const works2 = 'rSTAYKxF2K77ZLZ8GoAwTqPGaphAqMyXV'
+const works = 'r4MPhp7NeayAohgEd8FWSUV6eR2nLGwDX3'
+const no_toml2 = 'rV64miFA53xbWS3x9A6nRnzxMqLHN1t2h'
+const no_toml = 'rsoLo2S1kiGeCcn6hCUXVrCpGMWLrRrLZz'
+const no_domain = 'rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh'
+const wallet = no_toml
+let socket;
+
+
+function fetch_file_1(domain) {
+    // NODE AJAX equivalent
+    const url = "https://" + domain + TOML_PATH
+    console.log('Checking ' + url + '\n')
```

Comment:
Please use `makeLogEntry` (built in function for logging) instead

Reply:
>  if you could actually run the function and print out the output vs expected output

Would making the test file run three different tests with different outputs be the way to go about that? If not, can you give a bit more in-depth explanation? The test file is just leaving me confused.

Reply:
Yes making the test file run three different inputs and print out the output and the expected output would be a way to do it. so when the user runs the file, you would see a log message of " expecting error message  'Make sure you are entering a valid XRP address.\n', actual  'Make sure you are entering a valid XRP address.\n' " for the account not found case, so that the user can see clearly the test passed.